benches/src/lib.rs

```diff
@@ -165,7 +162,8 @@ impl VmBench {
 
         let mut db = new_db();
 
-        db.deploy_contract_with_id(&salt, &[], &contract, &state_root, &id)?;
+        db.deploy_contract_with_id(&salt, &[], &contract, &state_root, &id)
```

Comment:
It will be easier if you implement `impl From<fuel_core_storage::Error> for anyhow::Error`=) In this case, you don't need to update all functions related to `InterpreterStorage`